src/emqtt.erl

```diff
@@ -103,14 +111,13 @@
 
 -type(host() :: inet:ip_address() | inet:hostname()).
 
-%% Message handler is a set of callbacks defined to handle MQTT messages
-%% as well as the disconnect event.
```

Comment:
restore the comments ?

Reply:
It has been moved to the following lines so that it fits better